src/transaction/transaction_master_group_complete_manager.hpp

```diff
@@ -49,12 +49,14 @@ namespace cubtx
       static void final ();
 
       /* group_complete_manager methods */
-      void prepare_complete (THREAD_ENTRY *thread_p) override;
+      void do_prepare_complete (THREAD_ENTRY *thread_p) override;
       void do_complete (THREAD_ENTRY *thread_p) override;
 
       /* stream_ack methods */
       void notify_stream_ack (const cubstream::stream_position stream_pos) override;
 
+      int get_manager_type () const override;
```

Comment:
```suggestion
      log_tran_complete_manager_type get_manager_type () const override;
```

Reply:
Approved besides the suggestion regarding `get_manager_type ()` return type.

Reply:
This is part of CBRD-23203, where we may reconsider a little bit the actual design.  For now, I will add a comment.